internal/driver/cli.go

```diff
@@ -158,6 +158,37 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	return source, cmd, nil
 }
 
+// addBaseProfiles adds the list of base profiles or diff base profiles to
+// the source. This function will return an error if both base and diff base
+// profiles are specified.
+func (source *source) addBaseProfiles(flagBase, flagDiffBase []*string) error {
+	base := dropEmpty(flagBase)
+	diffBase := dropEmpty(flagDiffBase)
+
+	if len(base) > 0 && len(diffBase) > 0 {
+		return fmt.Errorf("-base and -diff_base flags cannot both be specified")
```

Comment:
When formatting is not needed, use errors.New()

Reply:
done.